src/deep-helloworld/frontend/bootstrap.js

```diff
@@ -41,4 +43,4 @@
       });
     });
   });
-})(DeepFramework);
+}(DeepFramework));
```

Comment:
Please undo to inner wrap, there are no sense to change. For more details please refer to: http://eslint.org/docs/rules/wrap-iife